pkgs/applications/video/devede/default.nix

```diff
@@ -1,18 +1,18 @@
-{ stdenv, fetchFromGitHub, python3Packages, ffmpeg_3, mplayer, vcdimager, cdrkit, dvdauthor
+{ stdenv, fetchFromGitLab, python3Packages, ffmpeg_3, mplayer, vcdimager, cdrkit, dvdauthor
 , gtk3, gettext, wrapGAppsHook, gdk-pixbuf, gobject-introspection }:
 
 let
   inherit (python3Packages) dbus-python buildPythonApplication pygobject3 urllib3 setuptools;
 
 in buildPythonApplication {
-  name = "devede-4.8.8";
```

Comment:
Please use pname + version.

Reply:
Thanks for the feedback! I'm using pname and version now.